sdk/core/core-amqp/samples/cbsAuth.ts

```diff
@@ -7,6 +7,7 @@ import {
   ConnectionConfig,
   CbsResponse
 } from "../src";
+import { TokenType } from "./../src/auth/token";
```

Comment:
Can we not combine this import line with the previous one? Same for other samples